Fix/#7123: Check for balance of EOA address on ethereum for checking its activity (#7136)
rotki/rotki: docs/changelog.rst

Changelog
=========

> * :bug:`7123` Now adding an EOA address that has only withdrawals/blocks activity will not fail.

Reviewer: ```suggestion
* :bug:`7123` Adding an EVM EOA address that has only withdrawals/blocks activity will no longer fail.
```
